Projects/Core/DesignSystem/Sources/TextField/SMSTextField.swift

```diff
@@ -0,0 +1,68 @@
+import SwiftUI
+
+public struct SMSTextField: View {
+    var placeholder: String
+    @Binding var text: String
+    var errorText: String
+    var isError: Bool
+    @FocusState var isFocused: Bool
+
+    public init(
+        _ placeholder: String = "",
+        text: Binding<String>,
+        errorText: String = "",
+        isError: Bool = false
+    ) {
+        self.placeholder = placeholder
+        _text = text
+        self.errorText = errorText
+        self.isError = isError
+    }
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            TextField(placeholder, text: $text)
+                .font(.sms(.body1))
+                .foregroundColor(.sms(.neutral(.n50)))
+                .padding(.horizontal, 12)
+                .padding(.vertical, 13.5)
+                .focused($isFocused)
+                .background(Color.sms(.neutral(.n10)))
+                .cornerRadius(8)
+                .overlay {
+                    RoundedRectangle(cornerRadius: 8)
+                        .strokeBorder(isFocused ? Color.sms(.primary(.p2)) : Color.white)
+                }
+                .modifier(SMSTextFieldClearModifier(text: $text))
+
+            if isError && !errorText.isEmpty {
+                Text(errorText)
+                    .font(.sms(.caption1))
+                    .foregroundColor(.sms(.system(.error)))
+                    .padding(.leading, 8)
+            }
+        }
```

Comment:
```suggestion
            if isError && !errorText.isEmpty {
                Text(errorText)
                    .font(.sms(.caption1))
                    .foregroundColor(.sms(.system(.error)))
                    .padding(.leading, 8)
                    .onTapGesture {
                        isFocused = true
                    }
            }
        }
```
로 placeholder의 텍스트를 눌렀을 때도 포커스 되게도 하죠

Reply:
넹

Reply:
어? 저기는 error 아닌가요?

Reply:
?

Reply:
그렇네

Reply:
뭐해!